Rakefile

```diff
@@ -27,6 +22,7 @@ end
 namespace :test do
   desc 'Run all development tests located in the test directory'
   Rake::TestTask.new :dev do |task|
+    task.options = RakeArguments.task_options
```

Comment:
Yeah, maybe `options` does make sense. The reason I went with `task_options` was to make it a little more clear to myself that it was the options for the task, not the options for rake.